assets/javascripts/job_templates.js

```diff
@@ -365,6 +365,65 @@ function toggleEdit() {
     checkJobGroupForm('#group_properties_form');
 }
 
+function toggleTemplateEditor() {
+    $('#media').toggle(250);
+    var form = $('#editor-form');
+    form.toggle(250);
+    form.find('.buttons').hide();
+    form.find('.progress-indication').show();
+    $('#toggle-yaml-editor').toggleClass('btn-secondary');
+    $('#editor-template').prop('readOnly', true).height($('#editor-yaml-guide').height());
```

Comment:
Reading the height (apparently) does not work for inline elements. I also would spell `readonly` lowercase for consistency.

I tested the following line and it works:
```
$('#editor-template').prop('readonly', true).height($('#editor-yaml-guide').parent().height());
```

Reply:
As per our conversation I'm going to switch it to a div so that should be okay